client/src/views/AllocationView/AllocationView.tsx

```diff
@@ -53,6 +53,7 @@ const AllocationView = (): ReactElement => {
   const { data: projectsEpoch } = useProjectsEpoch();
   const { data: projectsIpfsWithRewards } = useProjectsIpfsWithRewards();
   const { isRewardsForProjectsSet } = useAllocationViewSetRewardsForProjects();
+  const [isMultisigSignatureNeeded, setIsMultisigSignatureNeeded] = useState(false);
```

Comment:
Please rename to `isFirstMultisigSignatureSigned`, it would be easier to understand.

Instead of:
```
const [isMultisigSignatureNeeded, setIsMultisigSignatureNeeded] = useState(false);
const [isWaitingForWalletConfirmationMultisig, setIsWaitingForWalletConfirmationMultisig] =
    useState(false);
```
I suggest:

```
const [isFirstMultisigSignatureSigned, setIsFirstMultisigSignatureSigned] = useState(false);
const [areAllMultisigSignaturesSigned, setAreAllMultisigSignaturesSigned] =
    useState(false);
```